scripts/config/propogateAuroChange.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
```

Comment:
This document is a template, not actually useable in its current state? If that is the case, we should be more declarative about that.